pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1454,6 +1455,32 @@ def test_loc_bool_multiindex(self, dtype, indexer):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("utc", [False, True])
+    @pytest.mark.parametrize("column_label_array", [False, True])
```

Comment:
You can do ``@pytest.mark.parametrize("indexer", ["date", ["date]])``, then you don't need if on your test

Reply:
fixed